cpp/tests/spatial/point_distance_test.cpp

```diff
@@ -137,6 +137,6 @@ TEST_F(PairwisePointDistanceTestThrow, TypeMismatch)
   auto xy1     = fixed_width_column_wrapper<float>{1, 1, 2, 2, 3, 3};
   auto xy2     = fixed_width_column_wrapper<double>{1, 1, 2, 2, 3, 3};
 
-  EXPECT_THROW(pairwise_point_distance(xy1, offset1, xy2, offset2), cuspatial::logic_error);
+  EXPECT_THROW(pairwise_point_distance(offset1, xy1, offset2, xy2), cuspatial::logic_error);
```

Comment:
Did you intend to include all these argument reorderings in this PR?

Reply:
No, this change exists to revert changes that I made in this comment: https://github.com/rapidsai/cuspatial/pull/731#discussion_r991794161